apps/daimo-mobile/src/view/screen/send/SendTransferScreen.tsx

```diff
@@ -247,11 +276,27 @@ function SendConfirm({
   }
 
   const nav = useNav();
-
   const navToInput = () => {
     nav.navigate("SendTab", { screen: "SendTransfer", params: { recipient } });
   };
 
+  const rpcFunc = getRpcFunc(daimoChainFromId(account!.homeChainId));
+
+  // TODO: store homeCoin (foreignToken type) in account in the future
```

Comment:
calling it `homeCoin` and making the hardcoding explicit like this is a great way to do it